browser/components/WYSIWYGEditor.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import ProseMirror from 'react-prosemirror'
+import style from './WYSIWYGEditor.styl'
+import CSSModules from 'browser/lib/CSSModules'
+
+import 'prosemirror/dist/inputrules/autoinput'
+import 'prosemirror/dist/markdown'
+
+function buildStyle (fontSize) {
+  return {
+    fontSize: `${fontSize}px`
+  }
+}
+
+class WYSIWYGEditor extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      value: props.value,
+      config: props.config
+    }
+  }
+
+  componentWillReceiveProps (props) {
+    this.setState({
+      value: props.value,
+      config: props.config
+    })
+  }
+
+  handleOnChange (e) {
+    this.setState({
+      value: e
+    }, () => { this.value = e })
```

Comment:
Do you need 2nd argument? It seems that `this.value` is not used.

Reply:
Parent of this component(MarkdownNoteDetailComponent) is need `this.value`.
like below:
```
handleChange (e) {
  const { note } = this.state
  note.content = this.refs.content.value
  this.setState({note})
}
```